types/header.go

```diff
@@ -63,8 +62,8 @@ func (h *Header) ChainID() string {
 	return h.BaseHeader.ChainID
 }
 
-func (h *Header) Height() int64 {
-	return int64(h.BaseHeader.Height)
+func (h *Header) Height() uint64 {
```

Comment:
It is now uint64, as it should be. Would be nice to make Rollkit to use uin64 everywhere as well, but that's def not part of this PR.

Reply:
PR for it: https://github.com/rollkit/rollkit/pull/1177